src/pages/Dashboard/Markets/SupplyMarket/index.tsx

```diff
@@ -13,6 +13,7 @@ import { useTranslation } from 'translation';
 import { SupplyWithdrawModal } from '../../Modals';
 import { CollateralConfirmModal } from './CollateralConfirmModal';
 import { useStyles } from '../styles';
+import { useIsXlDown } from '../../../../hooks/responsive';
```

Comment:
We can use an absolute import here. It is nice if local imports are listed last. We could even use a plugin to enforce this

Reply:
done